vendor/vendor.json

```diff
@@ -50,6 +56,12 @@
 			"revision": "cd93ccfe0395e70031704ca68f14606588eec120",
 			"revisionTime": "2018-06-08T21:25:51Z"
 		},
+		{
+			"checksumSHA1": "/zjoJNfxpxVj22QUxBdBjQcsbHY=",
+			"path": "github.com/Azure/azure-sdk-for-go/services/servicebus/mgmt/2017-04-01/servicebus",
```

Comment:
Why are we updating/adding the Service package in this PR?

Reply:
I vendored the `azure` package from AzureRM, since it uses (for now) only the generic functions and it keeps getting out of sync. both of the new dependencies come from that package.